queries/cmake/highlights.scm

```diff
@@ -50,36 +59,67 @@
   (argument)* @parameter
 )
 
+(block_def
+  (block_command
+    (block) @function.builtin
+    (argument (unquoted_argument) @constant)
+    (#any-of? @constant "SCOPE_FOR" "POLICIES" "VARIABLES" "PROPAGATE")
```

Comment:
are those really constants? What's their value?

Reply:
These are a bit weird since CMake uses these "constants" to name parameters. Let's say if we have some like this in Lua
```lua
block{
SCOPE_FOR = {"POLICIES", "VARIABLES"},
PROPAGATE = {"var1", "var2"}
}
```
CMake would have something like this
```cmake
block(SCOPE_FOR POLICIES VARIABLES PROPAGATE var1 var2)
endblock()
```
So they are constants, but not really, however I can't find a more appropriate binding for these